packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -373,6 +373,9 @@ export default class CashierStore extends BaseStore {
     calculatePercentage(amount = this.converter_from_amount) {
         if (this.active_container === this.config.account_transfer.container) {
             this.percentage = +((amount / +this.config.account_transfer.selected_from.balance) * 100).toFixed(0);
+            if (!isFinite(this.percentage)) {
+                this.percentage = 0;
+            }
```

Comment:
can we put this outside of `if else`? so that it will also cover the issue in withdrawal

Reply:
sure!